scripts/build.sh

```diff
@@ -96,6 +100,22 @@ cp -r ./build/local-test-repo/org/opensearch "${OUTPUT}"/maven/org
 [ -z "$ARCHITECTURE" ] && ARCHITECTURE=$(uname -m)
 [ -z "$DISTRIBUTION" ] && DISTRIBUTION="tar"
 [ -z "$REVISION" ] && REVISION="1"
+[ -z "$BRANCH" ] && BRANCH="master"
+
+function download_template()
+{
+	echo "Downloading wazuh-template.json"
+	local download_url
+	download_url='https://raw.githubusercontent.com/wazuh/wazuh/$BRANCH/extensions/elasticsearch/7.x/'
+	curl -s ${download_url}wazuh-template.json -o distribution/src/config/wazuh-template.json
```

Comment:
Can't download as curl is not installed.

Reply:
`cURl` is added in #122.

For testing, it can be installed in the docker environment for this branch as follows:

```console
docker exec -it -u root wazuh-indexer-dev-4.9.0 bash 
aa078a753c16:/home/wazuh-indexer/app# apk add curl
```

Reply:
Also needs `git`. Can be added the same way as `curl`

Reply:
```
fatal: detected dubious ownership in repository at '/home/wazuh-indexer/app'
To add an exception for this directory, call:

        git config --global --add safe.directory /home/wazuh-indexer/app
+ GIT_COMMIT=
```